src/ImageSharp.Web.Providers.AWS/Caching/AWSS3StorageCache.cs

```diff
@@ -28,17 +29,21 @@ public AWSS3StorageCache(IOptions<AWSS3StorageCacheOptions> cacheOptions)
         AWSS3StorageCacheOptions options = cacheOptions.Value;
         this.bucketName = options.BucketName;
         this.amazonS3Client = AmazonS3ClientFactory.CreateClient(options);
+        this.cacheFolder = string.IsNullOrEmpty(options.CacheFolder)
+            ? string.Empty
+            : options.CacheFolder.Trim().Trim('/') + '/';
```

Comment:
Should I DRY this into a helper class, not to copy it between the two cache implementations?

Reply:
OK then.